scripts/prepare_conv_pp.py

```diff
@@ -0,0 +1,112 @@
+r"""
+Inputs
+-------
+in_path1 : str
+    path of input file with raw opsd data
+in_path2 : str
+    path of input file with geodata of regions in Germany
+out_path : str
+    path of output file with prepared data
+
+Outputs
+---------
+pandas.DataFrame
+    with grouped and aggregated data of conventional power plants in Berlin and Brandenburg.
+    Data is grouped by region, energy source, technology and chp capability and contains
+    net capacity and efficiency.
+
+Description
+-------------
+The script filters the OPSD conventional power plant package for power plants
+in Berlin and Brandenburg. The retrieved data is stored in a new dataframe, aggregated and
+saved as a csv file.
+Only operating power plants are considered.
+"""
+import sys
+import pandas as pd
+from os.path import dirname, abspath
+
+d = dirname(dirname(abspath(__file__)))
+
+sys.path.append(d)
+
+import oemof_b3.tools.geo as geo
+
+
+in_path1 = sys.argv[1]  # path to OPSD data
+in_path2 = sys.argv[2]  # path to geopackage of german regions
+out_path = sys.argv[3]
+
+if __name__ == "__main__":
+    opsd = pd.read_csv(in_path1)
+    b3 = opsd[opsd.state.isin(["Brandenburg", "Berlin"])]
+    b3 = b3.copy()
+    b3.reset_index(inplace=True, drop=True)
+
+    de = geo.load_regions_file(in_path2)
```

Comment:
```suggestion
    regions_nuts3_de = geo.load_regions_file(in_path2)
```

Reply:
More telling variable names are helpful.

Reply:
Not saying my suggestion is optimal, but tried to give more information in increasing order of specificity.